inc/views/base/member/emailuser.twig

```diff
@@ -0,0 +1,42 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.email_user }}</title>
+{% endblock head %}
+
+{% block body %}
+    {{ errors|raw }}
+    <form action="member.php" method="post" name="input">
+        <input type="hidden" name="my_post_key" value="{{ mybb.post_code }}" />
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td colspan="2" width="100%" class="thead"><strong>{{ lang.email_user }}</strong></td>
+            </tr>
+            {% if email.guest %}
+                <tr>
+                    <td width="40%" class="trow2"><strong>{{ lang.your_name }}</strong><br /><span class="smalltext">{{ lang.name_note }}</span></td>
+                    <td width="60%" class="trow2"><input type="text" class="textbox" size="50" name="fromname" value="{{ email.fromname }}" /></td>
+                </tr>
+                <tr>
+                    <td width="40%" class="trow1"><strong>{{ lang.your_email }}</strong><br /><span class="smalltext">{{ lang.email_note }}</span></td>
+                    <td width="60%" class="trow1"><input type="text" class="textbox" size="50" name="fromemail"  value="{{ email.fromemail }}" /></td>
+                </tr>
+            {% endif %}
+            <tr>
+                <td width="40%" class="trow2"><strong>{{ lang.email_subject }}</strong></td>
+                <td width="60%" class="trow2"><input type="text" class="textbox" size="50" name="subject" value="{{ email.subject }}" /></td>
+            </tr>
+            <tr>
+                <td valign="top" width="40%" class="trow1"><strong>{{ lang.email_message }}</strong></td>
+                <td width="60%" class="trow1"><textarea cols="50" rows="10" name="message">{{ email.message }}</textarea></td>
+            </tr>
+            {{ captcha|raw }}
```

Comment:
Captcha is fairly common, perhaps we should make a Twig function for it - thoughts?

Reply:
Definitely should be one.